src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days < 7) {
+    totalCost = totalCost - 20;
+
+    return totalCost;
```

Comment:
Here again, remove the `else` and just use an `if` with a `return`, as the `else` is not necessary due to the previous `return` statement (checklist item 5).